release/karaf/itests/camel/src/test/java/org/jboss/integration/fuse/karaf/itest/kiecamel/KieCamelJBPMWorkitemsQuickstartTest.java

```diff
@@ -0,0 +1,87 @@
+
+import static org.jboss.integration.fuse.karaf.itest.FeatureConstants.INTEG_PACK_FEATURE_ARTIFACT_ID;
+import static org.jboss.integration.fuse.karaf.itest.FeatureConstants.INTEG_PACK_QUICKSTARTS_FEATURE_GROUP_ID;
+import static org.ops4j.pax.exam.CoreOptions.maven;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.features;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import junit.framework.Assert;
+import org.jboss.integration.fuse.karaf.itest.KarafConfigUtil;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.Configuration;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.osgi.framework.BundleContext;
+
+/**
+ * Basic KIE-Camel with Blueprint and Decision tables functional tests running
+ * in Apache Karaf.
+ */
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class KieCamelJBPMWorkitemsQuickstartTest {
+
+    private static String FEATURE_NAME = "fuse-bxms-quickstart-camel-jbpm-workitems";
+
+    @Inject
+    protected BundleContext bundleContext;
+
+    @Configuration
+    public static Option[] configure() {
+        List<Option> options = new ArrayList<Option>();
+        Option[] defaultOptions = KarafConfigUtil.karafConfiguration();
+        options.addAll(Arrays.asList(defaultOptions));
+        options.add(features(maven().groupId(INTEG_PACK_QUICKSTARTS_FEATURE_GROUP_ID).artifactId(INTEG_PACK_FEATURE_ARTIFACT_ID).versionAsInProject().type("xml")
+                                 .classifier("features"), FEATURE_NAME));
+        options.add(org.ops4j.pax.exam.CoreOptions.bundle(org.ops4j.pax.exam.CoreOptions.mavenBundle().
+                                                          groupId("org.apache.httpcomponents").
+                                                          artifactId("httpclient-osgi")
+                                                          .version("4.3.6").getURL()));
+        options.add(org.ops4j.pax.exam.CoreOptions.bundle(org.ops4j.pax.exam.CoreOptions.mavenBundle().
+                                                          groupId("org.apache.httpcomponents").
+                                                          artifactId("httpcore-osgi").
+                                                          version("4.3.3").getURL()));
+        /*options.add(org.ops4j.pax.exam.CoreOptions.mavenBundle().
+                                                          groupId("org.ops4j.pax.web.itest").
+                                                          artifactId("pax-web-itest-base")
+        .version("4.2.5"))*/;
+
+        return options.toArray(new Option[1]);
+
+    }
+
+
+
+    @Test(timeout = 100000)
+    public void test() throws Exception {
+        Assert.assertTrue(true);
```

Comment:
OK, I guess I see the intent here, but this really should have a clear comment stating why the test does nothing currently.